migrations/sql/V2021.12.14.15.52__update_now_submission_data.sql

```diff
@@ -0,0 +1,42 @@
+ALTER TABLE now_submissions.application
+ADD COLUMN IF NOT EXISTS maxannualcubicmeters   numeric,
+ADD COLUMN IF NOT EXISTS proposedstartmonth     varchar(100),
```

Comment:
Just one overall comment, wondering if there is a special case to assign 100 or 300 length to day/month for the fields related to month and day. Would it be enough to add length 2 or 3 if you are handling days from 01-31 and months from 01-12?

Reply:
discussed on a call!